pkg/model/file.go

```diff
@@ -13,7 +13,71 @@
 
 package model
 
-import "path/filepath"
+import (
+	"os"
+	"path/filepath"
+)
+
+var (
+	// PipelineFiles represents the possible names an okteto pipeline can have
+	PipelineFiles = []string{
+		"okteto-pipeline.yml",
+		"okteto-pipeline.yaml",
+		"okteto-pipelines.yml",
+		"okteto-pipelines.yaml",
+		".okteto/okteto-pipeline.yml",
+		".okteto/okteto-pipeline.yaml",
+		".okteto/okteto-pipelines.yml",
+		".okteto/okteto-pipelines.yaml",
+	}
+	// ComposeFiles represents the possible names an okteto compose can have
+	ComposeFiles = []string{
+		"okteto-stack.yml",
+		"okteto-stack.yaml",
+		"stack.yml",
+		"stack.yaml",
+		".okteto/okteto-stack.yml",
+		".okteto/okteto-stack.yaml",
+		".okteto/stack.yml",
+		".okteto/stack.yaml",
+
+		"okteto-compose.yml",
+		"okteto-compose.yaml",
+		".okteto/okteto-compose.yml",
+		".okteto/okteto-compose.yaml",
+
+		"docker-compose.yml",
+		"docker-compose.yaml",
+		".okteto/docker-compose.yml",
+		".okteto/docker-compose.yaml",
+	}
+	// OktetoManifestFiles represents the possible names an okteto manifest can have
+	OktetoManifestFiles = []string{
+		"okteto.yml",
+		"okteto.yaml",
+		".okteto/okteto.yml",
+		".okteto/okteto.yaml",
+	}
+	// HelmChartsSubPaths represents the possible names a helm chart can have
+	HelmChartsSubPaths = []string{
+		"chart",
+		"charts",
+		"helm/chart",
+		"helm/charts",
+	}
+	// K8sManifestSubPaths represents the possible names a k8s manifest can have
+	K8sManifestSubPaths = []string{
```

Comment:
what about : **manifest, manifest.yml, manifest.yaml** ?

Reply:
they were not [considered before](https://github.com/okteto/okteto/pull/2835/files#diff-8d681e29798275482ee84496d80823cfe6c9a12b069672c92574ef39f82fcfd2L125), so I wouldn't include them as part of this PR.

Reply:
https://github.com/okteto/okteto/issues/2857